public/index.html

```diff
@@ -223,10 +223,8 @@
         </div>
       </nav>
       <!-- end top file bar -->
-      <div class="row container-fluid margin-0">
```

Comment:
This `container-fluid` I think wasn't doing anything so it's probably okay to not be here. But it could be kept as well.